pkg/endpoint/bpf.go

```diff
@@ -489,11 +489,19 @@ func (e *Endpoint) realizeBPFState(regenContext *regenerationContext) (compilati
 		// Compile and install BPF programs for this endpoint
 		if datapathRegenCtxt.regenerationLevel == RegenerateWithDatapathRebuild {
 			err = loader.CompileOrLoad(datapathRegenCtxt.completionCtx, datapathRegenCtxt.epInfoCache, &stats.datapathRealization)
-			e.getLogger().WithError(err).Info("Rewrote endpoint BPF program")
+			if err == nil {
+				e.getLogger().Debug("Rewrote endpoint BPF program")
+			} else {
+				e.getLogger().WithError(err).Warning("Error while rewriting endpoint BPF program")
+			}
 			compilationExecuted = true
 		} else { // RegenerateWithDatapathLoad
 			err = loader.ReloadDatapath(datapathRegenCtxt.completionCtx, datapathRegenCtxt.epInfoCache, &stats.datapathRealization)
-			e.getLogger().WithError(err).Info("Reloaded endpoint BPF program")
+			if err == nil {
+				e.getLogger().Debug("Reloaded endpoint BPF program")
```

Comment:
Even this one should be pretty rare (albeit, on older kernels happening periodically due to ipcache GC replacing the datapath map).

Reply:
Changed it back to Info level.